### ISSUE TITLE
Black screen when connecting to Desktop - macOS Catalina (10.15.1)

### ISSUE THREAD

**Issue by GnaXi** [Ylianst/MeshCentral#667]
Hi,

I've installed the MeshCentral Agent on a Mac computer running macOS Catalina (10.15.1) and when I try to connect to the desktop I just get a black screen.
Terminal and Files connect work fine.

I've also made sure that it has the accessibility permission and record screen permission in macOS.

**Comment by krayon007** [comment]
This will take some time to troubleshoot, because my MacOS dev system got hosed when I tried to upgrade to Catalina. But I'll look into this when it's back up and running.

**Comment by MailYouLater** [comment]
Related/Duplicate: #473, #576.

**Comment by johnnyq** [comment]
yes same here

**Comment by GnaXi** [comment]
@krayon007 I can get you remote access to a macOS system with Catalina, if that would be any help :)

**Comment by GnaXi** [comment]
Interestingly enough, I just installed the Agent on another machine running Catalina, this time an older MacBook Air.
On this machine I am actually able to see the entire screen and also able to move the mouse, but I am not able to click or drag anything around.

**Comment by GnaXi** [comment]
Another update regarding my above comment.
After the installation of the MeshCentral Agent and you request access to the desktop, macOS prompts you to give meshagent_osx64 access to Screen Recording, which I gave it.
Screen Recording access is not enough to be able to actually control the system.

I rebooted the MacBook Air and requested Desktop access again and this time it prompts me for Accessibility, which is what is needed to actually control the machine.
After giving it this permission I was also able to control it.

As for as I know, Accessibility is enough to both record and control the machine. It's at least the only permission that is requested by TeamViewer.
